src/util/glTF/glTF2Loader.js

```diff
@@ -64,23 +173,23 @@ x3dom.glTF2Loader.prototype._traverseNodes = function(node, parent)
  * Generates a X3D node from a glTF node
  * @param {Object} node - A glTF-Node
  */
-x3dom.glTF2Loader.prototype._generateX3DNode = function(node)
+x3dom.glTF2Loader.prototype._generateX3DNode = function(node, parent, index)
 {
     var x3dNode;
 
     if( node.matrix != undefined )
     {
         x3dNode = this._generateX3DMatrixTransform(node);
     }
-    else if( node.translation != undefined || 
-             node.rotation    != undefined ||
-             node.scale       != undefined)
-    {
-        x3dNode = this._generateX3DTransform(node);
-    }
+//     else if( node.translation != undefined || 
```

Comment:
There is a lot of temporary stuff. This in particularly is about to just always use Transform nodes (as glTF node equivalents) since it is both simpler and required if the node is a target for an animation. So the commenting means that the code will be likely removed.
Testing with any glTF which uses animation (but not for skinning/morphing) would be the most useful review activity.